src/test/ensemble_flow_custom_node_tests.cpp

```diff
@@ -3290,6 +3290,16 @@ TEST_F(EnsembleFlowCustomNodeAndDynamicDemultiplexerLoadConfigThenExecuteTest, J
     this->checkResponse("pipeline_output", response, expectedOutput, {dynamicDemultiplyCount, 1, 10});
 }
 
+TEST_F(EnsembleFlowCustomNodeAndDynamicDemultiplexerLoadConfigThenExecuteTest, DynamicDemultiplexerNoResults) {
```

Comment:
Maybe put this test just before DISABLED_JustDynamicDemultiplexerConfigReturning0Batch?

Reply:
Done